builder/files/erigon.service

```diff
@@ -0,0 +1,34 @@
+[Unit]
+  Description=erigon
+  StartLimitIntervalSec=500
+  StartLimitBurst=5
+
+[Service]
+  Restart=on-failure
+  RestartSec=5s
+  ExecStart=/usr/local/bin/erigon \
```

Comment:
Do you have to specify the `bindir` for nfpm or does it default to this location

Reply:
manually copy erigon binary to this dir in dockerfile.release here: https://github.com/maticnetwork/erigon/pull/3/files#diff-701366a9e47be6026c2b3a9cc9484a02f319b9b8469b28e907ac1237fe2456caR5